Add passive node (#4008)
influxdata/docs-v2: content/enterprise_influxdb/v1.9/tools/influxd-ctl.md

Added data node 3 at cluster-data-node:8088
```

###### Add a passive node to a cluster

> **Passive nodes** are data nodes that do not own any shards. Passive nodes have the ability to acts as load balancers. They accept write calls, perform shard lookup and RPC calls and distribute writes to the data node that owns the shard. If you are using passive nodes, they should be the write endpoint for all data ingest. A cluster can have multiple passive nodes.

Reviewer: ```suggestion
**Passive nodes** act as load balancers--they accept write calls, perform shard lookup and RPC calls (on active data nodes), and distribute writes to active data nodes. They do not own shards or accept writes. If you are using passive nodes, they should be the write endpoint for all data ingest. A cluster can have multiple passive nodes.
```
